app/components/publications/publication/proofs/proof-edit-modal.hbs

```diff
@@ -35,12 +35,14 @@
                   @onChange={{this.setReceivedAtDate}} />
               </div>
             {{/if}}
-            <div class="auk-form-group">
-              <Auk::Label for="proofUploadIsCorrected">{{t "corrected"}}</Auk::Label>
-              <Auk::Checkbox id="proofUploadIsCorrected"
-                @checked={{this.isCorrected}}
-                @disabled={{true}} />
-            </div>
+            {{#if this.isCorrected}}
+              <div class="auk-form-group">
+                <Auk::Label for="proofUploadIsCorrected">{{t "corrected"}}</Auk::Label>
+                <Auk::Checkbox id="proofUploadIsCorrected"
+                  @checked={{this.isCorrected}}
```

Comment:
Op zich niet fout, `@checked={{this.isCorrected}}` zou in dit geval altijd `true` moeten zijn, zetten we dan hier dan ook beter `@checked={{true}}`?